pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -28,6 +28,37 @@ Depending on the cluster size, it will take a few minutes to be created. After t
 dkp get kubeconfig -c ${CLUSTER_NAME} > ${CLUSTER_NAME}.conf
 ```
 
+### Confirming `calico` installation
+
+Before exploring the new cluster further, you will want to confirm your `calico` installation is correct.
```

Comment:
```suggestion
Before exploring the new cluster further, you will want to confirm your `calico` installation is correct.
In the default case, Calico will autodetect the IP to use for each node using the the `first-found` [method](https://projectcalico.docs.tigera.io/reference/node/configuration#ip-autodetection-methods). This may not be appropriate in our particular nodes and you will need to modify Calico's configuration to use a different method.
One approach is to use the `interface` method by providing the interface id to use. For example lets assume that all your cluster nodes use `ens192` as the interface name, follow the steps outlined in this section to modify Calico configuration. 

```

Reply:
Applied those changes @dkoshkin - review again, please?